README.md

```diff
@@ -475,20 +475,20 @@ Supported Locales
 * ar (Armenian)
```

Comment:
how about making this section collapsible? 
Not sure that everyone what to scroll all the locales mentioned here to see what else is in readme